scss/_patterns_links.scss

```diff
@@ -6,9 +6,12 @@
     }
 
     // Used for links point at a different domain
-    &--external::after {
-      @include link-svg-icon;
-      margin-left: .25em;
+    &--external {
+      @include link-svg-icon($color-link);
```

Comment:
This will need to be on an after element or a background image as the content is replacing the content of the link.